tests/unit/pubsub.tcl

```diff
@@ -423,4 +423,9 @@ start_server {tags {"pubsub network"}} {
         assert_equal "pmessage * __keyevent@${db}__:new bar" [$rd1 read]
         $rd1 close
     }
+
+    test {PUBSUB HELP should not have unexpected options} {
```

Comment:
we actually want to non-error response...

can you please gather all HELP tests in one test function?

also, please add
```
client|help 
cluster|help 
command|help 
config|help 
function|help
```

something like
```
assert_match "*CLIENT <subcommand> *" [r CLIENT HELP]
```